browserdebuggertools/wssessionmanager.py

```diff
@@ -147,7 +124,7 @@ def run(self):
                 self._empty_send_queue()
                 self._empty_websocket()
 
-                self._send_queue.wait_for_messages()
+                time.sleep(self._POLL_INTERVAL)
                 self._last_poll = time.time()
```

Comment:
It does not look like you need to update `self._last_poll` before you enter the while loop

Reply:
I think we do, say the very first recv operation hangs we'd never have a value for `self._last_poll` and the ws will never be considered blocked. As for the check that `self._last_poll` is set in `blocked` we need that too just in case the thread fails to get to that point in which there would be an exception on the thread. But things could be a little clearer/optimal here.